samples/cfgs/models/heads/linear_cls.yaml

```diff
@@ -1,6 +1,6 @@
 model:
     head:
-        type: LinearClsHead
+        type: CustomLinearClsHead
```

Comment:
Wouldn't it be better to create a new 'custom_linear_cls.yaml' file? (The same goes for non_linear.)